manifests/domain.pp

```diff
@@ -25,8 +25,8 @@
 # * Trevor Vaughan <mailto:tvaughan@onyxpoint.com>
 #
 define sssd::domain (
-  $id_provider,
-  $debug_level = '',
+  String $id_provider,
+  String $debug_level = '',
   $debug_timestamps = '',
```

Comment:
![Data type all the params!](https://cdn.meme.am/Instance/Preview?imageID=1121885&generatorTypeID=&panels=&text0=data%20type&text1=all%20the%20params!)